### DISCUSSION TITLE
250bp read length compatibility DB

### DISCUSSION THREAD

**Discussion by Sanrrone** [bhattlab/phanta#24] [Ideas]
Dears,
It would be great resource a DB compatible for 250bp read length (which is almost 150bp assembled paired-reads). I think including this length would cover all the lengths for illumina since there are already 100bp and 150bp.

**Comment by yipinto** [comment]
Sure!
Will do.
Just to make sure, do you plan to use Phanta to profile shotgun metagenomics/virome sequencing data? I'm asking because 250x2 is not a common read length for these types of data.

**Reply by Sanrrone** [reply]
Actually I have 150x2 and thats why I am getting around ~250 (as a singleton). But, at the same time is usefull for those who use this kind of length

**Reply by yipinto** [reply]
So why not to run it in paired-end mode, 150x2?

**Reply by Sanrrone** [reply]
I am following an internal protocol that use the merged reads for better results (but yeah, at the beginning I look for the same, just using the 150x2).

**Reply by yipinto** [reply]
Hi @Sanrrone,
You can download the 250bp Bracken, for the default database, from here: http://ab_phanta.os.scg.stanford.edu/Phanta_DBs/database250mers.kmer_distrib
We will support that read length from now on.

**Reply by Sanrrone** [reply]
awesome! thanks you very much
